Sources/TSPL/TSPL.docc/ReferenceManual/Statements.md

```diff
@@ -819,6 +819,31 @@ This means that a `defer` statement can be used, for example,
 to perform manual resource management such as closing file descriptors,
 and to perform actions that need to happen even if an error is thrown.
 
+The `defer` statement is lexically scoped and executes at the end of the scope
+in which it is declared.
```

Comment:
Not sure about "declared" here. Statements aren't declared. "scope in which it is stated"?

Reply:
I think you're mostly defining what it means to be lexically scoped here, so rather than "and", you want something like:

> The `defer` statement is lexically scoped; that is, its contents run at the end of the scope that encloses the `defer` statement.

Reply:
Okay, re-addressed;

> The *statements* in the `defer` statement are executed at the end of the scope
 enclosing the `defer`.

Reply:
@krilnon, are you okay with the changes?

Reply:
Yes, looks good now.